tests/console/kubeadm.pm

```diff
@@ -19,24 +19,20 @@ use utils "systemctl";
 sub run {
     select_console("root-console");
 
-    record_info 'etcd', 'Stop etcd and clean up';
-    systemctl 'disable --now etcd';
-    script_run 'rm -r /var/lib/etcd/*';
-
-    record_info 'Setup', 'Test: Package Installation';
-    my $packages = 'kubernetes-client kubernetes-kubelet kubernetes-kubeadm docker-kubic cri-tools';
-    trup_install($packages);
+    record_info 'Test #1', 'Test: Initialize kubeadm';
+    assert_script_run('kubeadm init --cri-socket=/var/run/crio/crio.sock --pod-network-cidr=10.244.0.0/16', 180);
 
-    record_info 'Prepare', 'Test: Enable and start required services';
-    systemctl('enable --now kubelet docker');
-    systemctl('is-active docker');
-    systemctl('is-active kubelet');
+    record_info 'Test #2', 'Test: Configure kubectl';
+    assert_script_run('mkdir -p ~/.kube');
+    assert_script_run('cp -i /etc/kubernetes/admin.conf ~/.kube/config');
```

Comment:
is there any specific reason you use the `-i` interactive flag?

Reply:
because it's in the docu...